person.rb

```diff
@@ -31,4 +32,8 @@ def can_use_services?
   def correct_name
     @name
   end
+
+  def add_rental(rental)
+    @rentals << rental
+  end
```

Comment:
- [ ] good job adding the `add_rental` method, however, the method should create a new instance of Rental, have a look at [the example in a previous lesson](https://github.com/microverseinc/curriculum-ruby/blob/main/oop/articles/oop_relationships_by_examples.md#many-to-many-relationship). you can achieve this by doing the following:
```
  def add_rental(book, date)
    Rental.new(date, book, self)
  end

```